### PR TITLE
Desktop features

### PR DESCRIPTION
**Pull request by mwismann**
### Portfolio Project Desktop Version
**In this branch I worked on the following features:**

- Add semantic content to the HTML file for the desktop version.
- Add the corresponding media query breakpoint.
- Styled the CSS file to apply changes on the desktop version.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -331,33 +345,293 @@ input,
...
+
+  .headline {
+    margin-top: 0;
+    width: 100%;
+    height: 100vh;
+    background: url(icon-img/desktop/Header-illustration-desktop.svg) no-repeat;
+    background-size: 100%;
+    justify-content: flex-start;
+    gap: 2.5rem;
+  }
```
Comment (iambenkis) [inline styles.css:420]: - [x] Greet job @mwismann , I really never see a design beautifully made like this , however according to the requirement you should stick as much as possible to the Figma design, though make sure that the missing details on this picture are completed 

### Figma design 

Screenshot from 2023-01-20 00-22-53


### Your design

Screenshot from 2023-01-20 00-23-41

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @mwismann 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (mwismann) [reply styles.css:420]: Hi @iambenkis ,

I really appreciate your words and advice! 🙏

I just have a question regarding the design. As you can see in the image below, the background image works for me, even in different screen sizes 😖 

### **Design on my browser**
image

### **With Inspect**
image

**Review by bimbolabuari — COMMENTED** [review]
Hello @mwismann  👋,

_This is an additional review as a result of the second opinion request._

- Currently, all background images are appearing as expected in your application.

https://user-images.githubusercontent.com/66112760/213754560-3a98a1cb-f3b8-4fd1-9d09-2423c08a6564.mov



As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀! //

**Review by binhussen — APPROVED** [review]
Hi @mwismann , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**